docs/classes/user_stats.md

```diff
@@ -194,10 +194,12 @@ Provides functions for accessing and submitting stats, achievements, and leaderb
 !!! function "getAchievementIcon( ```string``` name )"
 	Gets the icon for an achievement.
 
-	Triggers a [user_achievement_icon_fetched](#user_achievement_icon_fetched) callback.
+	The image is returned as a handle to be used with [getImageRGBA](utils.md#getimagergba) to get the actual image data.
```

Comment:
Good additions; however this function does trigger a callback and should be noted.

Reply:
The link didn't work for me, so I looked around for it, and couldn't find any documentation about the signal. I expanded my search to include the `godotsteam.cpp` file and couldn't find it anywhere in there, either. Eventually I found a note in the GodotSteam 3.x changelogs (version 3.3) that the callback was removed because it was never called.

Reply:
Haha, then you are more thorough than I was!  OK, in that case then it should be removed.